beacon_node/network/src/sync/block_lookups/mod.rs

```diff
@@ -512,7 +512,8 @@ impl<T: BeaconChainTypes> BlockLookups<T> {
 
         let action = match result {
             BlockProcessingResult::Ok(AvailabilityProcessingStatus::Imported(_))
-            | BlockProcessingResult::Err(BlockError::BlockIsAlreadyKnown(_)) => {
+            | BlockProcessingResult::Err(BlockError::DuplicateFullyImported(..))
+            | BlockProcessingResult::Err(BlockError::DuplicateImportStatusUnknown(..)) => {
                 // Successfully imported
```

Comment:
This comment may be outdated now?

Reply:
Asked for input on this from LH network gang

Reply:
I have a `TODO` on this to add a comment saying this is unreachable because we never send RPC blocks for gossip verification, so we can't hit `DuplicateImportStatusUnknown`